### ISSUE TITLE
Auto scaling has no animation and is not smooth

### ISSUE THREAD

**Issue by Tamuna** [PhilJay/MPAndroidChart#4654]
when I set `isAutoScaleMinMaxEnabled = true` it does it's business, but it is not good for eye. I would be happy if I could set animation to scaling.

**Comment by Kazachenko1998** [comment]
I completely agree, I would like to be able to implement this using library tools.

**Comment by AAEvstefeev** [comment]
@Kazachenko1998  and so? You are manage it?
